package.json

```diff
@@ -7,8 +7,8 @@
 		"scssfiles": "src/*.scss src/**/*.scss"
 	},
 	"engines": {
-		"node": ">=16.0.0",
-		"npm": ">=8 <9"
+		"node": ">=20.10.0",
+		"npm": ">=10.2.3"
```

Comment:
This configuration matches the one used in Gutenberg ([reference](https://github.com/WordPress/gutenberg/blob/1275d2f0d02961a8cc109bcc3f3ab76a8609bdd6/package.json#L17-L20)).